src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -50,11 +46,25 @@ public Actor clone() {
     }
 
     @Override
-    public String toString() {
-        return "Actor{"
-                + "id=" + id
-                + ", name='" + name + '\''
-                + ", country='" + country + '\''
-                + '}';
+    public boolean equals(Object obj) {
```

Comment:
do you really need equals and hashCode? 
if yes, let's use Lombok annotations
same comment for other places

Reply:
it's not recommended to use Lombok equals/hashcode annotation for entity class
![image](https://github.com/user-attachments/assets/8d4b4d12-a8fa-4009-a533-4bd3bd279eb6)